geckomat/limit_proteins/flexibilizeProteins.m

```diff
@@ -149,7 +149,7 @@
     %If protein was flexibilized set its upper bound to the simulated
     %concentration
     if ismember(name,flexProts)
-        if optSolution(index)>0
+        if optSolution(index)>0 && optSolution(index)>abundances(i)
```

Comment:
@edkerk good find! any reason to keep the `optSolution(index)>0` condition? I tested locally removing it and the final result is the same, if you want I can add that commit here. Otherwise everything looks good!

Reply:
I didn't want to modify the code too much to prevent unwanted behaviour, but now I've had a critical look at that line and the `optSolution(index)>0` part is indeed obsolete.